net/remote/remote-node.sh

```diff
@@ -263,7 +263,7 @@ EOF
       agave-ledger-tool -l config/bootstrap-validator shred-version --max-genesis-archive-unpacked-size 1073741824 | tee config/shred-version
 
       if [[ -n "$maybeWaitForSupermajority" ]]; then
-        bankHash=$(agave-ledger-tool -l config/bootstrap-validator bank-hash --halt-at-slot 0)
+        bankHash=$(agave-ledger-tool -l config/bootstrap-validator verify --halt-at-slot 0 --print-bank-hash | jq -r .hash)
```

Comment:
Also change
`jq -r .hash`
to
`jq -r ".hash"`

Reply:
Did this error out at you ? Testing locally with a sample json file `test.json`:
```
{
    "slot": 123456789,
    "hash": "asdfasdfasdfasdf"
}
```
I see no difference between `jq -r .hash` and `jq -r ".hash"`:
```
$ cat test.json | jq -r .hash  
asdfasdfasdfasdf

$ cat test.json | jq -r ".hash"
asdfasdfasdfasdf

// Not using -r keeps the quotation marks
$ cat test.json | jq .hash     
"asdfasdfasdfasdf"
```

Reply:
Yes when the quotes were missing, my GCE machines didn't give the correct result. Maybe it's because of different jq versions?

Reply:
Interesting, the fact that it is executing as part of a script might matter too (vs. me running `jq` in a terminal directly). In any case, testing is the source of truth so I made the change in https://github.com/anza-xyz/agave/pull/1745/commits/ddb78bb81758eb121b0e27304d6970e6b899c984

Reply:
Great, thanks!

Reply:
Of course - thank you very much for doing the testing!